automation/canary-prod-test.sh

```diff
@@ -123,10 +123,17 @@ if [ $ENV = "prod" ]; then
     done
 
     if [ "$submissionStatus" == "Done" ] && [ "$workflowsStatus" == "Succeeded" ]; then
-      echo "One-off workflow finished within 5 minutes with workflow status: $workflowsStatus"
+      timer=$SECONDS
```

Comment:
won't this always be 0? i think you can do `start=$SECONDS` at the top and then subtract the difference down here

Reply:
No, it works I have runs with 208 and 206 seconds that correspond to the actual run times.